YangMingShan/YMSPhotoPicker/Public/YMSPhotoPickerViewController.m

```diff
@@ -213,6 +214,12 @@ - (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPa
     if (indexPath.row == 0) {
         [self yms_presentCameraCaptureViewWithDelegate:self];
     }
+    else if (self.shouldReturnAssetForSingleSelection) {
```

Comment:
How about moving this condition checking under line 223 ?
This would prevent user enable multipleSelection and return Asset/Image at the same time.